frontend/src/app/modules/matches/matches-computations.js

```diff
@@ -15,6 +15,23 @@ export const generateMatchRatingChanges = (userId, userMatches) => userMatches.m
     }
 })
 
+export const generateRatingHistoryGraphForUser = (userMatches, userId, initialRating) => userMatches
```

Comment:
I think `generateRatingHistoryGraphForUser` may be too long 🤔

Reply:
I think it's perfectly fine and descriptive